lib/node_modules/@stdlib/array/ones/docs/types/index.d.ts

```diff
@@ -216,7 +52,7 @@ declare function ones( length: number, dtype: 'generic' ): Array<number>;
 * var arr = ones( 2, 'float32' );
 * // returns <Float32Array>[ 1.0, 1.0 ]
 */
-declare function ones( length: number, dtype?: DataType ): Float64Array;
+declare function ones<T extends keyof DataTypeMap<number> = 'float64'>( length: number, dtype?: T ): DataTypeMap<number>[T];
```

Comment:
This should be `NumericAndGenericDataTypeMap`.

Reply:
Changed.